src/runtime/storage/storage.ts

```diff
@@ -38,6 +38,7 @@ type HandleOptions = {
 type ArcLike = {
   generateID: () => Id;
   idGenerator: IdGenerator;
+  getActiveStore: <T extends Type>(storeInfo: StoreInfo<T>) => ToStore<T>;
```

Comment:
is this why you plumbed active stores through manifest.ts? It should be getStoreInfo instead, and in order to activate you'll need extra contextual stuff that doesn't then end up in manifests.

Reply:
yes, refactored all callers to additionally pass storageService instead.